stable/aad-pod-identity/README.md

```diff
@@ -0,0 +1,127 @@
+# aad-pod-identity
+
+[aad-pod-identity](https://github.com/Azure/aad-pod-identity) enables Kubernetes applications to access cloud resources securely with [Azure Active Directory](https://azure.microsoft.com/en-us/services/active-directory/) (AAD).
+
+## TL;DR:
+
+```console
+$ helm install stable/aad-pod-identity -f values.yaml
+```
+
+## Introduction
+
+A simple [helm](https://helm.sh/) chart for setting up the components needed to use [Azure Active Directory Pod Identity](https://github.com/Azure/aad-pod-identity) in Kubernetes.
+
+This helm chart will deploy the following resources:
+* AzureIdentity `CustomResourceDefinition`
+* AzureIdentityBinding `CustomResourceDefinition`
+* AzureAssignedIdentity `CustomResourceDefinition`
+* AzureIdentity instance (optional)
+* AzureIdentityBinding instance (optional)
+* Managed Identity Controller (MIC) `Deployment`
+* Node Managed Identity (NMI) `DaemonSet`
+
+## Getting Started
+The following steps will help you create a new Azure identity ([Managed Service Identity](https://docs.microsoft.com/en-us/azure/active-directory/managed-identities-azure-resources/overview) or [Service Principal](https://docs.microsoft.com/en-us/azure/active-directory/develop/app-objects-and-service-principals)) and assign it to pods running in your Kubernetes cluster.
```

Comment:
Since `AzureIdentity instance` is optional, maybe this section should be optional?

Reply:
Thanks @ritazh. Done!